### DISCUSSION TITLE
Tab as UserControl not working

### DISCUSSION THREAD

**Discussion by dhunjoshi** [flet-dev/flet#720] [Q&A]
import flet as ft
from flet import *

class TabDemo(UserControl):

    def build(self):
        t = ft.Tabs(
            selected_index=1,
            animation_duration=300,
            tabs=[
                ft.Tab(
                    text="Tab 1",
                    content=ft.Container(
                        content=ft.Text("This is Tab 1"), alignment=ft.alignment.center
                    ),
                ),
                ft.Tab(
                    tab_content=ft.Icon(ft.icons.SEARCH),
                    content=ft.Text("This is Tab 2"),
                ),
                ft.Tab(
                    text="Tab 3",
                    icon=ft.icons.SETTINGS,
                    content=ft.Text("This is Tab 3"),
                ),
            ],
            expand=1,
        )
        return t

def main(page: ft.Page):

    page.add(TabDemo())

ft.app(target=main)

**Comment by FeodorFitsner** [comment]
Please see a solution here: https://github.com/flet-dev/flet/issues/703#issuecomment-1355932549
